src/main/java/exceptions/InvalidBoardPositionException.java

```diff
@@ -0,0 +1,7 @@
+package exceptions;
+
+public class InvalidBoardPositionException extends RuntimeException {
```

Comment:
Is it necessary to have to separate exception classes for an invalid position?

Reply:
No, you are right but I think that we have to refactor the class name Position to BoardPosition